src/routes.py

```diff
@@ -21,12 +23,13 @@ def callback():
     if token:
         try:
             cache_data = process_token(token)
-            print("Processed Token Data:", cache_data)
+            logging.info("Processed Token Data: %s", cache_data)
             return jsonify({"status": "success", "message": "Token received", "data": cache_data})
         except ValueError as e:
             logging.error('Cannot process_token %s', e)
+            opaque_token_count = get_cache(OPAQUE_TOKEN_COUNT_KEY)
             opaque_token = get_cache(f'opaque_{opaque_token_count}')
-            print("Opaque Token:", opaque_token)
+            logging.info("Erroneous Token: %s", opaque_token)
```

Comment:
This isn't an erroneous token, it's just a normal opaque token.

For info logging, remove the erronous.